packages/components/src/MultiSelect/subcomponents/MultiSelectToggle/MultiSelectToggle.tsx

```diff
@@ -1,6 +1,8 @@
 import React, { HTMLAttributes, forwardRef } from "react"
 import classnames from "classnames"
 import { ClearButton } from "~components/ClearButton"
+// import { ClearButton } from "~components/ClearButton"
```

Comment:
Should this be here?

Reply:
Not anymore, just a hangover of a rebase, will 🪓